fendermint/eth/api/src/mpool.rs

```diff
@@ -21,11 +22,12 @@ use crate::{cache::Cache, state::Nonce, HybridClient};
 
 const RETRY_SLEEP_SECS: u64 = 5;
 
+pub type SignedTransaction = (TypedTransaction, et::Signature);
 /// Cache submitted transactions by their Ethereum hash, because the CometBFT
 /// API would not be able to find them until they are delivered to the application
 /// and indexed by their domain hash, which some tools interpret as the transaction
 /// being dropped from the mempool.
-pub type TransactionCache = Cache<et::TxHash, et::Transaction>;
```

Comment:
Honestly, now that I look at the final solution, I think this was better the way it was before. We ended up leaking all of ethers' type complexity into Tendermint. In fact, I don't think TypedTransaction should've leaked anywhere at all. I'll work on rolling this back, removing `to_eth_typed_transaction`, `to_eth_legacy_request`, `to_eth_eip1559_request`, etc.

Reply:
I feel the `conv::*` should not be in `message` crate. It should be shifted to `eth` crate. That might be better.